### PR TITLE
Feature/donghoon/add negative prompt

### PR DESCRIPTION
**Pull request by 589hero**
- Negative prompt 추가
- minor) diffusion pipeline load 부분 수정

### CODE REVIEW

File: src/schemas.py [hunk 1]
```diff
@@ -10,6 +10,10 @@ class ImageGenerationRequest(BaseModel):
         ...,
         description="try adding increments to your prompt such as 'oil on canvas', 'a painting', 'a book cover'",
     )
+    negative_prompt: Optional[str] = Field(
+        None,
```
Comment (byeongal) [inline src/schemas.py]: `None` -> ""

File: src/schemas.py [hunk 2]
```diff
@@ -10,6 +10,10 @@ class ImageGenerationRequest(BaseModel):
...
+        description="add image feature that you want to remove on your generated image",
+    )
     steps: int = Field(
         default=45, ge=1, le=100, description="more steps can increase quality but will take longer to generate"
```
Comment (byeongal) [inline src/schemas.py]: `45` -> `50`

**Review by byeongal — APPROVED** [review]
LGTM with minor comments

**Comment by 589hero** [comment]
Thank you reviewing!

**Pull request merged**